.jenkins/cscs-perftests/batch.sh

```diff
@@ -49,48 +47,11 @@ configure_build_errors=0
 test_errors=0
 plot_errors=0
 
-hpx_targets=("future_overhead_report_test" "stream_report_test")
-hpx_test_options=("--hpx:queuing=local-priority --hpx:threads=4 --test-all \
-    --repetitions=100 --futures=500000" \
-    "--vector_size=1048576 --hpx:threads=4 --iterations=5000 \
-    --warmup_iterations=500")
-
-# Build binaries for performance tests
-${perftests_dir}/driver.py -v -l $logfile build -b release -o build \
-    --source-dir ${src_dir} --build-dir ${build_dir} -e $envfile \
-    -t "${hpx_targets[@]}" \
-    || { echo 'Build failed'; configure_build_errors=1; exit 1; }
-
-index=0
-result_files=""
-
-# Run and compare for each targets specified
-for executable in "${hpx_targets[@]}"
-do
-  test_opts=${hpx_test_options[$index]}
-  result=${build_dir}/reports/${executable}.json
-  reference=${perftests_dir}/perftest/references/daint_default/${executable}.json
-  result_files+=(${result})
-  references_files+=(${reference})
-  logfile_tmp=log_perftests_${executable}.tmp
-
-  run_command=("./bin/${executable} ${test_opts}")
-
-  # TODO: make schedulers and other options vary
-
-  # Run performance tests
-  ${perftests_dir}/driver.py -v -l $logfile_tmp perftest run --local True \
-      --run_output $result --targets-and-opts "${run_command[@]}" \
-      || { echo 'Running failed'; test_errors=1; exit 1; }
-
-  index=$((index+1))
-done
+# Synchronize after the asynchronous copy from the source dir
+wait
```

Comment:
I learned about `wait` from here, very cool! I wonder if it would be clearer to put this closer to the backgrounded copies? Or does the virtualenv `activate` take a long time so it's worth it to put it after it?

Reply:
yes it's quite long in my experience, but I don't mind so much if you prefer having it before